### ISSUE TITLE
Add Control Flow Graph visualization to the playground.

### ISSUE THREAD

**Issue by rzvxa** [oxc-project/playground#9]
I hope we can adopt [this](https://github.com/dreampuf/GraphvizOnline) project for this purpose, It's been my go-to visualizer, and having it in the playground would be a nice to have feature.

###### I've self-assigned it but if anyone decides to pick it up sooner feel free to do so, and announce it in this issue before starting to work on it.

**Comment by IWANABETHATGUY** [comment]
Here is biome cfg visualization, 
![image](https://github.com/oxc-project/playground/assets/17974631/2572be90-5b92-437c-a264-264d188155f2)
https://biomejs.dev/playground/?code=ZgB1AG4AYwB0AGkAbwBuACAAdABlAHMAdAAoACkAIAB7AAoAIAAgAGwAZQB0ACAAYQAgAD0AIAAxADAAOwAKAAoAIAAgAGkAZgAgACgAYQApACAAewAKACAAIAAgACAAYwBvAG4AcwBvAGwAZQAuAGwAbwBnACgAJwB0AGUAcwB0ACcAKQAKACAAIAB9ACAAZQBsAHMAZQAgAHsACgAgACAAIAAgAGMAbwBuAHMAbwBsAGUALgBsAG8AZwAoACcAZABmAGoAYQBpAGYAagBlAGkAbwBhAGoAZgBpAG8AYQB3AGoAaQBvACcAKQAKACAAIAB9AAoAfQA%3D

**Comment by IWANABETHATGUY** [comment]
They use `mermaid` to display the control flow graph** https://github.com/rome/tools/pull/2969/files
